doc/release/release_0.14.rst

```diff
@@ -19,7 +21,7 @@ Bug Fixes
 - Fix potential use of NULL pointer (#3696)
 - pypi: explicitly exclude Python 3.1, 3.2, and 3.3 (#3726)
 - Reduce default tolerance in threshold_li (#3622) (#3781)
-- Denoising functions now accept float32 images (#3449) (#3486) (#3880)
+- Denoising functions now accept float32 images (#3449) (#3486) (#3880) (#3936)
```

Comment:
@stefanv Denoising functions were still rejecting float32 images. We think we finally got the final nail on the coffin with #3936.

Since it was a major bugfix of the release, @jni and I were thinking it would be appropriate to include it in this "next" patch release.